src/ride/controllers/search_ride_controller.py

```diff
@@ -13,8 +13,7 @@
 
 
 class SearchRideController:
-    def __init__(self, data, request, serializer):
```

Comment:
are we removing this because we changed it from a POST to a GET endpoint

Reply:
yes